packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import * as KeyCode from 'keycode-js';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import onClickOutside from 'react-onclickoutside';
```

Comment:
Adding this as dependency will fix this import lint error.

Reply:
Added in f8e6d07